solr/core/src/test/org/apache/solr/rest/schema/TestBulkSchemaAPI.java

```diff
@@ -66,16 +66,6 @@ public void before() throws Exception {
 
     createJettyAndHarness(
         tmpSolrHome, "solrconfig-managed-schema.xml", "schema-rest.xml", "/solr", true, null);
-    if (random().nextBoolean()) {
```

Comment:
The v2 Schema APIs now differ from v1 by more than just the path prefix, so this quick-and-dirty randomization no longer works.